HSLdevcom/jore4-ui
# Stop table row priority

### StopAreaDetails: Fix hardcoded stings to be translations


### Make react happy with the Toast element - noCamelDataAttrs
Data attributes should-be-defined-with-dashes.


### Add top level Suspense support
Adds a high level fallback spinner for React Suspense feature. A more
appropriate spinner should be defined on a lower level closer to where
a suspenseful child component is used.


### Add Suspense support to LoadingWrapper


### StopSearch: Replace LoadingStopsRow with LoadingWrapper
Essentially this change adds Suspense support on a correct level to
grouped Stop Search results.


### StopTableRow: Implement PriorityTd

- - -

### How to test
1. Go to http://localhost:3300/stop-registry/search?query=*
2. Click the open on map button from one of the  result rows
3. Modify the Stop on the map, and set it to be a temporay stop
4. Close map
5. Row visualization should be changed,
6. Open map again and create a draft stop.
7. Use the search page to find that stop, it should have draft icon on it.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/HSLdevcom/jore4-ui/946)
